tests/forte/data/multi_pack_test.py

```diff
@@ -176,6 +176,15 @@ def test_multipack_entries(self):
             ],
         )
 
+        # fix bug 559: additional test for index to pack_id changes
```

Comment:
This test probably does not test the compatibility part? We should probably put two example serialized packs (with different versions) there and load them, make sure we get what we want.

Reply:
This is done in the latest check-in of "entry_data_structures_test.py" -- a new unit test "test_mp_pointer_with_version" was added using old (non-versioned) string to deserialize and test on using version and using the index instead of (new) pack_id when the version is old.